### PR TITLE
 #beSR33eo-Log out authenticated User

### PR DESCRIPTION
**Pull request by Mugema123**
**What does this PR do?**

-Authenticated users should be able to logout.

**description of tasks to be completed**
-Authenticated users are able to logout.
-Appropriate error messages are displayed on unsuccessful attempts
-Skeleton UIs/screen are/is shown when the page loads/is loading.
-The app layouts are displayed
-The page is responsive.
**How can be manually tested?**
-clone this repo
-run npm install
-run npm run dev
-login in with email (Josh@gmail.com) and Password(admin)
 
-Log out

**Relavant trello story**
# beSR33eo

### CODE REVIEW

**Review by TuyizeeAnastase — COMMENTED** [review]
Screen Shot 2021-01-12 at 09 14 54
i can't login

**Review by TuyizeeAnastase — APPROVED** [review]

**Comment by Ally4** [comment]
can put it on the logged-in landing page with the other button on navigation, I suggest you work with Josh as he did the login?.

**Review by Ally4 — APPROVED** [review]

**Review by ceelogre — COMMENTED** [review]
I'm unable to login. Could you add the login details to pr description.

**Comment by TuyizeeAnastase** [comment]
login credentials:
```
email:"Josh@gmail.com"
password:"admin"
```

File: src/tests/logout.test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import '@testing-library/jest-dom'
+import { render } from "@testing-library/react";
+import logoutReducer from '../redux/reducers/reducerLogout'
+import { logoutAction, logoutRequest, logoutSuccess, logoutError } from '../redux/actions/authLogout';
+import { Provider } from "react-redux";
+// import { LOGOUT_REQUEST, LOGOUT_SUCCESS, LOGOUT_ERROR } from '../redux/types/logoutTypes,
```
Comment (Ntwali-Josue) [inline src/tests/logout.test.js]: check these comments and remove them, they're unnecessary

File: src/tests/logout.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    it('Capturing Snapshot of Logout', () => {
+        const renderedValue = LogoutComponent()
+        expect(renderedValue).toMatchSnapshot();
+    });
+
+    it('should test action', ()=>{
+        expect(logoutRequest()).toHaveProperty('type')
+    })
+
+    it('should test action', ()=>{
```
Comment (Ntwali-Josue) [inline src/tests/logout.test.js]: maybe you can try and give different names to these test titles, since they don't have the same assertions.

**Review by Ntwali-Josue — APPROVED** [review]

**Pull request merged**